example/pretty.cpp

```diff
@@ -112,6 +112,9 @@ pretty_print( std::ostream& os, json::value const& jv, std::string* indent = nul
         break;
 
     case json::kind::double_:
+        // write double values with full precision
+        // see https://stackoverflow.com/questions/554063/how-do-i-print-a-double-value-with-full-precision-using-cout)
+        os.precision(std::numeric_limits<double>().digits10);
         os << jv.get_double();
```

Comment:
this should be using the library function to output the number

Reply:
That would work

Reply:
No